test/services/oauth2/hydra.js

```diff
@@ -73,63 +73,62 @@ describe('oauth2 service', function oauthTest() {
 
 	before((done) => {
 		this.timeout(10000);
-		Promise.all([
-			toolService.create(testTool1),
-			toolService.create(testTool2),
-		]).then(() => {
-			clientsService
-				.create(testClient2)
-				.then((client) => {
-					const oauth = oauth2.create({
-						client: {
-							id: client.client_id,
-							secret: client.client_secret,
-						},
-						auth: {
-							tokenHost: 'http://localhost:9000',
-							tokenPath: '/oauth2/token',
-							authorizePath: '/oauth2/auth',
-						},
-					});
-					const authorizationUri = oauth.authorizationCode.authorizeURL(
-						{
-							redirect_uri: client.redirect_uris[0],
-							scope: 'openid',
-							state: '12345678',
-						},
-					);
-					request({
-						uri: authorizationUri,
-						method: 'GET',
-						followRedirect: false,
-					}).catch((res) => {
-						const position =								res.error.indexOf('login_challenge=')
-								+ 'login_challenge'.length
-								+ 1;
-						loginRequest1 = res.error.substr(position, 32);
+		toolService.create(testTool1).then(() => {
+			toolService.create(testTool2).then(() => {
```

Comment:
Should be possible to run it in paralelle, because you could not prefent, that two users doing the action parallel